docs-13183 acm 250 known issue

JennyHajee opened the pull request:
Added instructions for known issue fix of manually adding the admin profile when upgrading to 2.5.0

JennyHajee committed:
docs-13183 acm 250 known issue

Added instructions for known issue fix of manually adding the admin profile when upgrading to 2.5.0

ferferrarazzo commented on modules/ROOT/pages/update-acm.adoc:
```diff
@@ -29,6 +29,12 @@ Portal user availability enables portal users in the same customer or partner po
+.. Add the new Member Management tab to Api Curator.
+.. Click *Interface Setup* > *User Management Settings* > *Disable Enhanced Profile User Interface*, change the *User Management Settings*, and disable the *Enhanced Profile User*.
+.. Click *Setup* > *Profiles* > *Edit*, scroll to *Custom Tab Settings* > *Member Management* > *Default On*, and *Save*.
+.. Click *Interface Setup* > *User Management Settings* > *Disable Enhanced Profile User Interface* and enable *Enhanced Profile User*.
```
```suggestion
.. Click *Interface Setup* > *User Management Settings* > enable *Enhanced Profile User Interface*.
```

JennyHajee replied:
Thanks, I will make this change.

JennyHajee committed:
Update update-acm.adoc

Moved Manually updated admin profile from step 3 to step 1 and edited line 26 per Fer's comment.

JennyHajee commented:
commits were not verified so created another PR
